docs/source/Plugin/P093.rst

```diff
@@ -127,6 +127,12 @@ Special thanks
 Change log
 ----------
 
+.. versionchanged:: 2023/09/25
+  ...
+
+  |added|
+  Set Remote Temperature (greater than 0.0 external Temperature Sensor is used. equal 0.0 internal Temperature Sensor is used.)
```

Comment:
Had to read this multiple times before it hit me what you tried to tell here.
Maybe the column width in this Git diff view isn't helping...

Anyway, maybe this is easier to grasp:

```
Set Remote Temperature. (value > 0: Use external temperature sensor, value = 0: Use internal temperature sensor)
```